cmd/uninstall.go

```diff
@@ -48,10 +48,14 @@ which is used to uninstall tiup.
   tiup uninstall --all`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if self {
-				if err := os.RemoveAll(env.Profile().Root()); err != nil {
-					return errors.Trace(err)
+				deletable := []string{"bin", "manifest", "components"}
```

Comment:
so the packages download by `cluster` are still there, how about just not delete anything by default, and prompt user an option to delete everything?